### PR TITLE
release 1.1-8

### PR DESCRIPTION
**Pull request by pengdake**
1.[init][update] fix some problem about tools self update
2.[template] update nova profile for soft-delete
3.[manage][volume] support to delete private type volume
4.[manage][volume] support non-interactive mode to delete volume
5.[Bugfix][doctor][cls] print "checking haproxy resource"
6.添加 eayunstack cleanup orphan 功能
7.Fix redmine #6524
8.Fix issue #224
9.[bugfix]fix redmine #7317
10.fix redmine 7390
11.[feature]filter max_pool_size in confuguration files
12.Fix `eayunstack -h` run on fuel ERROR

Signed-off-by: pq 19921207pq@gmail.com

### CODE REVIEW

**Comment by blkart** [comment]
**[属于1.1-7]** 1.[init][update] fix some problem about tools self update
**[属于1.1-7]** 2.[template] update nova profile for soft-delete
**[属于1.1-7]** 3.[manage][volume] support to delete private type volume
**[属于1.1-7]** 4.[manage][volume] support non-interactive mode to delete volume
**[属于1.1-7]** ]5.[Bugfix][doctor][cls] print "checking haproxy resource"
6.添加 eayunstack cleanup orphan 功能
7.Fix redmine #6524
8.Fix issue #224
9.[bugfix]fix redmine #7317
10.fix redmine 7390
11.[feature]filter max_pool_size in confuguration files  **redmine #7433**
12.Fix eayunstack -h run on fuel ERROR

---
- 不要把以前版本的更新内容写进来
- 有 redmine 编号的尽量把编号写上

File: eayunstack-tools.spec [hunk 1]
```diff
@@ -78,6 +78,9 @@ fi
 
 
 %changelog
+* Wed May 25 2016 blkart <blkart.org@gmail.com> 1.1-8
```
Comment (blkart) [inline eayunstack-tools.spec]: 这一行有问题

File: eayunstack-tools.spec [hunk 2]
```diff
@@ -78,6 +78,9 @@ fi
...
+- commit 9f5c33bdb400d3a6ff3711d524027f5911164c0e
```
Comment (blkart) [inline eayunstack-tools.spec]: 不是 master 分支最后一个 commit id

File: eayunstack-tools.spec (hunk 1 above)
Comment (pengdake) [inline eayunstack-tools.spec]: 最上面commit的日期不是最新的啊，所以我用的第二个，这个怎样处理啊

File: eayunstack-tools.spec (hunk 2 above)
Comment (pengdake) [inline eayunstack-tools.spec]: 同上

File: eayunstack-tools.spec [hunk 3]
```diff
@@ -78,6 +78,9 @@ fi
 
 
 %changelog
+* Wed Apr 20 2016 pq     <19921207pq@gmail.com> 1.1-8
+- commit 3c20e3f2ba8bd017f8f9147fc7ef4420a325be3d
```
Comment (blkart) [inline eayunstack-tools.spec]: 最后一个 commit id 不是 3da26e77ff8724eb22d35fd47723ca684d3afd8c 吗？

**Pull request merged**